### ISSUE TITLE
Deploying Jenkins HOL contains bunch of low resolution images

### ISSUE THREAD

**Issue by sa2sa2** [oracle/learning-library#1573]
Deploying Jenkins HOL contains bunch of low resolution images :

https://oracle.github.io/learning-library/oci-library/DevOps/Deploying_Jenkins/Deploying_Jenkins_HOL.html

which makes the HOL hard to follow because of these low resolution images, it would be beneficial if the HOL pictures are clear enough so that it can be followed easily.

I would recommend Oracle to review all the OCI HOLs images quality, since this issue is noticed in more than one HOL.

**Comment by tmcginn** [comment]
We moved a number of OCI workshops to LiveLabs: http://bit.ly/golivelabs

This isn't one we added yet, but I will add this to the list and update the images accordingly.
